metrics/network/remote_network/README.md

```diff
@@ -34,3 +38,19 @@ $ cd metrics/network/remote_network
 $ bash remote-networking-iperf3.sh -b -i eno1 -u tester -a 10.xxx.xxx.xxx
 
 ```
+
+## Running the `remote-networking-memory-smem.sh` test
+
+`remote-networking-memory-smem.sh` should run in host A. The script needs the following inputs to run:
+- Arguments of the test to run. In this case the argument is `-r` for RSS memory.
```

Comment:
can we inject a line with the more 'formal' command line syntax notation here, a bit like you have down in the example code around line 54 - just to make it clearer these are 4 arguments that need to go in a set order? I think that will make it clearer up front, and save having to read down to the actual example to see the syntax.